app/services/payments/process_credit_purchase.rb

```diff
@@ -25,8 +25,14 @@ def initialize(user, credits_count, purchase_options:)
     end
 
     def call
-      process_purchase
-      create_credits if success?
+      # we expect at least one of :stripe_token, :organization_id, or :selected_card to process
+      if purchase_options.compact_blank.present?
```

Comment:
Since organization id is passed in the `purchase_options` hash but not used during purchase (only after purchase in `create_credits`) we need to either slice `:stripe_token, :selected_card` here or just check at least one of those are non-blank.